Form/Type/BasePickerType.php

```diff
@@ -30,12 +31,23 @@
      */
     private $formatConverter;
 
+    protected $translator;
+
+    protected $locale;
+
     /**
      * @param MomentFormatConverter $formatConverter
      */
-    public function __construct(MomentFormatConverter $formatConverter)
+    public function __construct(MomentFormatConverter $formatConverter, TranslatorInterface $translator = null)
     {
         $this->formatConverter = $formatConverter;
+        $this->translator = $translator;
+
+        if ($this->translator !== null) {
+            $this->locale = $this->translator->getLocale();
```

Comment:
Nice catch.

Null translator argument should be deprecated.

Please notify users with a `trigger_error`